Formula/vcpkg.rb

```diff
@@ -0,0 +1,29 @@
+class Vcpkg < Formula
+  desc "C++ Library Manager for Windows, Linux, and MacOS"
+  homepage "https://docs.microsoft.com/en-us/cpp/vcpkg"
+
+  stable do
+    url "https://github.com/microsoft/vcpkg/archive/2019.08.tar.gz"
+    sha256 "d533d8b577ce527c018aaaa77cee88d1940e3772221b7a6b4b995d3e953730e1"
+  end
+
+  head do
+    url "https://github.com/microsoft/vcpkg.git"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
```

Comment:
Does it not build with the system compiler (clang/LLVM)?

Reply:
It seems `vcpkg` 2019.08 used some C++ features that system clang/LLVM doesn't support yet.

```
./bootstrap-vcpkg.sh -useSystemBinaries -allowAppleClang
Last 15 lines from /Users/fl0120/Library/Logs/Homebrew/vcpkg/01.bootstrap-vcpkg.sh:
In file included from ../src/vcpkg/archives.cpp:3:
In file included from ../include/vcpkg/archives.h:3:
../include/vcpkg/base/files.h:96:29: error: 'path' is unavailable: introduced in macOS 10.15
void is_directory(const fs::path& p, std::error_code& ec) = delete;
                            ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/filesystem:739:24: note: 'path' has been explicitly marked unavailable here
class _LIBCPP_TYPE_VIS path {
                       ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
```